src/data/markdown/docs/01 guides/03 Results visualization/CSV.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'CSV'
+excerpt: ''
+---
+
+You can also make k6 output detailed statistics in a CSV format by using the `--out/-o` option for `k6 run`, like this:
+
+<div class="code-group" data-props='{"labels": ["CLI"]}'>
+
+```shell
+$ k6 run --out csv=my_test_result.csv script.js
+```
+
+</div>
+
+Or if you want to get the result gzipped, like this:
+
+<div class="code-group" data-props='{"labels": ["CLI"]}'>
+
+```shell
+$ k6 run --out csv=my_test_result.gz script.js
+```
+
+</div>
+
+## CSV format
+
+The CSV result file will contain lines like these:
+
+<div class="code-group" data-props='{"labels": ["Output"]}'>
+
+```csv
+metric_name,timestamp,metric_value,check,error,error_code,group,method,name,proto,scenario,status,subproto,tls_version,url,extra_tags
+http_reqs,1595325560,1.000000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+http_req_duration,1595325560,221.899000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+http_req_blocked,1595325560,225.275000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+http_req_connecting,1595325560,217.680000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+http_req_tls_handshaking,1595325560,0.000000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+http_req_sending,1595325560,0.112000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+http_req_waiting,1595325560,220.280000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+http_req_receiving,1595325560,1.507000,,,,,GET,http://test.k6.io,HTTP/1.1,default,200,,,http://test.k6.io,
+vus,1595325560,1.000000,,,,,,,,,,,,,
+vus_max,1595325560,20.000000,,,,,,,,,,,,,
+checks,1595325561,1.000000,status is 200,,,,,,,default,,,,,
+checks,1595325561,0.000000,response body,,,,,,,default,,,,,
+data_sent,1595325561,76.000000,,,,,,,,default,,,,,
+data_received,1595325561,11045.000000,,,,,,,,default,,,,,
+iteration_duration,1595325561,1449.049580,,,,,,,,default,,,,,
+iterations,1595325561,1.000000,,,,,,,,default,,,,,
+```
+
+</div>
+
+Each entry in the report represents a metric `metric_name` along with it's value `metric_value` at time `timestamp`. If error, then the `error` along with the `error_code` is captured.
```

Comment:
Maybe link to https://k6.io/docs/javascript-api/error-codes as well ?

Reply:
Done